### PR TITLE
Update README and other docs

### PR DESCRIPTION
**Pull request by nprogers-WPE**
# What Are We Doing Here

Updated README and other docs to include logo. Added table of contents. Reorganized and reworded docs.

## How to Verify

1. Check out this PR
2. Run Command X, Click Button Y
3. Profit

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
+![Lostrómos logo](docs/images/logo.png)
 # Contribution Guidelines
 
-## Security issues
+## <a name="bugs"></a>Report a Bug
```
Comment (wpdrush21) [inline CONTRIBUTING.md]: What does this end up doing? My html/markdown is weak in this area, so I'm not sure how that is different from

## Report a Bug

File: docs/deployment.md [hunk 2]
```diff
@@ -1,18 +0,0 @@
-# Deployment
-
-We don't deploy a docker image with Lostrómos, but instead only release the binary to github so that it can be built in
-to the image you need for your particular application. This is due to the link between kubectl and Lostromos, and it's
-possible the kubectl we intend to use isn't valid for everyones use case. We do build a docker image as part of testing
-that can be thought of as an [example](../test/docker/Dockerfile) however.
-
-## GOOS Environment Variable
```
Comment (wpdrush21) [inline docs/deployment.md]: Where does this end up getting moved to? Is it cut out now?

File: README.md [hunk 3]
```diff
@@ -1,83 +1,52 @@
...
-
-## Contributing
-
-See [Contribution Guildelines](./CONTRIBUTING.md) to get started.
-
-## Report a Bug
-
-To report an issue or suggest an improvement please open an [issue](https://github.com/wpengine/lostromos/issues).
\ No newline at end of file
+WP Engine uses Lostrómos in conjunction with another tool to customize VM deployments into GCE. Each VM instance offloads some of its workload to a separate Kubernetes application. As we create new VMs in GCE, this other tool monitors the Google API for these changes and creates a CR as they happen. Lostrómos watches for changes to this CR endpoint and creates a Helm release by combining information from the new CR and a predefined template. This allows us to deconstruct some of the work for our deployment into GCE and reduce maintenance work around sharing the data between applications.
```
Comment (wpdrush21) [inline README.md]: Link `WP Engine` to our site for some promotion?

File: README.md [hunk 4]
```diff
@@ -1,83 +1,52 @@
-# Lostrómos
+![Lostrómos logo](docs/images/logo.png)
 
 [![Build Status](https://travis-ci.org/wpengine/lostromos.svg?branch=master)](https://travis-ci.org/wpengine/lostromos)
 [![codecov](https://codecov.io/gh/wpengine/lostromos/branch/master/graph/badge.svg)](https://codecov.io/gh/wpengine/lostromos)
 
-**NOTE**: Under active development. Not ready for production usage.
```
Comment (lvaughn) [inline README.md]: 👍

File: CONTRIBUTING.md (hunk 1 above)
Comment (nprogers-WPE) [reply CONTRIBUTING.md]: It's so I can reference the tag from another page.

File: docs/deployment.md (hunk 2 above)
Comment (nprogers-WPE) [reply docs/deployment.md]: Yes. I cut some stuff for clarity with the idea of circling back.

**Pull request closed without merging**